modules/graphstudio/pages/zoomChart-licenseExpiry-Upgrade.adoc

```diff
@@ -0,0 +1,116 @@
+= ZoomChart License Expiry and Deprecation Upgrade Instructions
+
+== Overview
+TigerGraph's license to use and distribute the ZoomCharts visualization library embedded within certain versions of GraphStudio will expire *December 31, 2023*.
+New versions of the TigerGraph Suite which *does not* use ZoomCharts has been developed and was released with TigerGraph 3.9.3.
+
+== Upgrade GraphStudio
+
+*We encourage you to perform an  upgrade as soon as possible to take advantage of the latest features and improvements in TigerGraph Suite.*
+Failure to upgrade before the expiry date may result in the display of a "License Expired" label in your graph visualizations.
+
+NOTE: No other features will be affected and this upgrade will not affect production use or result in any downtime for the upgrading system.
```

Comment:
I'm double-checking if we need to restart any service (i.e. GUI) after the new bins are applied

cc @samzhou2

Reply:
@OmarAyo1 @JosephNewmanTG  No restart will be needed, it's a hot upgrade.